lib/khmer.pc.in

```diff
@@ -0,0 +1,13 @@
+prefix=@prefix@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+sharedlibdir=${libdir}
+includedir=${prefix}/include
+
+Name: khmer
+Description: khmer cpp library
```

Comment:
-> "The unsupported core C++ library from the khmer project"

+

`URL: http://khmer.readthedocs.org/`